### PR TITLE
Files and folders renamed from norwegian to english

### PR DESCRIPTION
**Pull request by yleaxeman**
Renaming file and folder names, not example content or README files content.

### CODE REVIEW

File: src/app/core/mock-db.service.ts [hunk 1]
```diff
@@ -2,13 +2,13 @@ import { Injectable } from '@angular/core';
...
 import { AngularNgbEksemplerData } from 'src/mock-db-data/library-angular/angular-ngb-eksempler.data';
-import { AngularFontAwesomeEksemplerData } from 'src/mock-db-data/library-angular/angular-font-awesome-eksempler.data';
+import { AngularFontAwesomeExamplesData } from 'src/mock-db-data/library-angular/angular-font-awesome-eksempler.data';
 
-import { CssSidemalerData } from 'src/mock-db-data/library-css/css-sidemaler.data';
-import { CssModulerData } from 'src/mock-db-data/library-css/css-moduler.data';
-import { CssKomponenterData } from 'src/mock-db-data/library-css/css-komponenter.data';
-import { CssFargerSkriftData } from 'src/mock-db-data/library-css/css-farger-skrift.data';
-import { CssIkonerData } from 'src/mock-db-data/library-css/css-ikoner.data';
+import { CssSidemalerData } from 'src/mock-db-data/library-css/css-page-templates.data';
```
Comment (proand) [inline src/app/core/mock-db.service.ts:7]: Should we go all the way while we're at it, even though PR are named "Files and folders..." ?

**Review by proand — COMMENTED** [review]
See inline comment

File: src/app/core/mock-db.service.ts (hunk 1 above)
Comment (yleaxeman) [reply src/app/core/mock-db.service.ts:7]: Definitely

**Review by proand — APPROVED** [review]
LGTM 👍

**Pull request merged**